lib/inspec/metadata.rb

```diff
@@ -35,14 +35,9 @@ def initialize(ref, logger = nil)
     end
 
     def supports(sth, version = nil)
-      params[:supports] ||= []
```

Comment:
Actually not quite: We load yaml and then only finalize the params. That function is never called :)

Look:
* https://github.com/chef/inspec/pull/367/files#diff-0bdee087faa99fbed7459cc5302e01efL141
* https://github.com/chef/inspec/pull/367/files#diff-0bdee087faa99fbed7459cc5302e01efL133